### ISSUE TITLE
Move the ... button in palettes so that they don't overlap with the scroll bar

### ISSUE THREAD

**Issue by FrancRos31** [musescore/MuseScore#25267]
### Issue type

UI bug (incorrect info or interface appearance)

### Description with steps to reproduce

1. open some palettes to enable the scroll bar;
2. try to open a palette.

Actual behaviour: you have to dodge the scroll bar while you search for the ... button.
Expected behaviour: scroll bar and ... button should never overlap.

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/624e95e0-54f3-4a99-81b1-6d307b302ac0

### What is the latest version of MuseScore Studio where this issue is present?

OS: Windows 10 Version 2009 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.4.2-242570931, revision: github-musescore-musescore-3130f97

### Regression

I was unable to check

### Operating system

Windows 10 Pro

### Additional context

Looks like the already closed #12402.
Solving this issue would make the workflow more fluid.

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by krasko78** [comment]
Workaround: right-click on the palette header instead. :)

**Comment by FrancRos31** [comment]
Thanks, lol. Does it mean you're gonna close this? Maybe something in the future can be done.

**Comment by krasko78** [comment]
No, I am not closing it. I said merely as an FYI. Besides, I am not one of the bosses around here, lol.

**Comment by bkunda** [comment]
@jessjwilliamson could you please make a quick specification for how this should be improved?

**Comment by jessjwilliamson** [comment]
> @jessjwilliamson could you please make a quick specification for how this should be improved?

This scrollbar is just too wide - 8 pixels. Changing this to 6 pixels wide instead should solve the issue :)
 
Original UI designs we have are large (8 pixels) small (6 pixels) and score (with stroke outline). 
We have designs that use 6 pixel scrollbars but it looks like they're not being used in the app. I think we need to review our use of scrollbars throughout the whole app soon. I can see a lot of 8 pixel wide ones and the score one but barely any 6. Tagging @avvvvve as we had a similar conversation about this not too long ago.

**Comment by krasko78** [comment]
Not sure that simply decreasing the width of the scrollbar to 6px solves the problem:
![image](https://github.com/user-attachments/assets/9e7bde4c-4b08-4ca5-8e7e-8f9f1e9cfce6)
